dashboard/static/js/controllers.js

```diff
@@ -26,18 +28,59 @@ visApp.controller('mainController', ['$scope', 'Get','$location', '$anchorScroll
             $anchorScroll();
         });
     }
+    // get the user default selection from DB and if not available selects the first item from the dropdown list
+    get_default_selection = function () {
+        let get_defaults_url = '/api/v1/courses/' + dashboard.course_id + '/get_user_default_selection?default_type=assignment';
+        let re = Get.getData(get_defaults_url)
```

Comment:
same thing: is "let" necessary here?

Reply:
I had an email hanging around in draft about this for awhile, let is the more appropriate thing to use here.